src/main/java/com/j256/ormlite/dao/Dao.java

```diff
@@ -30,7 +31,7 @@
 /**
  * The definition of the Database Access Objects that handle the reading and writing a class from the database. Kudos to
  * Robert A. for the general concept of this hierarchy.
- * 
+ *
```

Comment:
Can you fix these space removals.  My IDE will put them back if I format the class.